### PR TITLE
ENH/FIX?: Drop support for Python 2.6

### PR DESCRIPTION
**Pull request by chrisfilo**
Python 2.7 brings argparse and some useful testing functions
(http://stackoverflow.com/questions/17585207/python-unittest-assetraises).
Both debian stable and ubuntu LST come with Python 2.7.

### CODE REVIEW

File: doc/users/install.rst [hunk 1]
```diff
@@ -107,7 +107,7 @@ recommendations.
 Must Have
 ~~~~~~~~~
 
-Python_ 2.6 - 2.7
+Python_ 2.7
 
 Nibabel_ 1.0 - 1.4
```
Comment (satra) [inline doc/users/install.rst]: should we update the ranges for the rest of the libraries?

Comment (chrisfilo) [inline doc/users/install.rst]: yes - but let's do this in a separate PR
